deps/internal/cmds/debug/debug.go

```diff
@@ -0,0 +1,85 @@
+package debug
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/depscloud/depscloud/deps/internal/client"
+	"github.com/depscloud/depscloud/internal/mux"
+
+	"github.com/spf13/cobra"
+)
+
+func Command(version mux.Version) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "debug",
+		Short: "Output information helpful for debugging",
+		RunE: func(_ *cobra.Command, args []string) error {
+			// Printing Client environment variables
+			systemInfo := client.GetSystemInfo()
+			fmt.Println(systemInfo.String())
+
+			// Printing Client version information
+			versionString := fmt.Sprintf("Client Version: %s", version)
+			fmt.Println(versionString)
+
+			debugClient := httpDebugClient{client: http.DefaultClient, baseURL: systemInfo.BaseURL}
+
+			// Printing Server version information
+			serverVersion, err := debugClient.GetServerVersion()
+			if err != nil {
+				fmt.Println(fmt.Sprintf("Error While retrieving server version"))
+			} else {
+				fmt.Println(fmt.Sprintf("Server Version: %s", serverVersion))
+			}
+			// Printing Server Health information
+			healthString, err := debugClient.GetHealth()
+			if err != nil {
+				fmt.Println(fmt.Sprintf("Error While retrieving server health"))
+			} else {
+				fmt.Println(fmt.Sprintf("Server Health: %s", healthString))
+			}
+			return nil
```

Comment:
So I think this would read better if it followed the flow of:
* fetch information
* Output information

Reply:
Changed it to the mentioned flow

Reply:
Changed it to the above mentioned flow